docs/concepts/model-monitoring.md

```diff
@@ -32,9 +32,14 @@ Model monitoring is currently in Tech Preview status. Migrating from 1.7.0 to la
 
 ```{toctree}
 :maxdepth: 1
-
-../model-monitoring/index
-../model-monitoring/model-monitoring-flow
-../model-monitoring/monitoring-models
 ../model-monitoring/applications
+../model-monitoring/monitoring-models
+../model-monitoring/monitoring-models-grafana
+../model-monitoring/index
+../concepts/alerts
 ```
+
+**See also**
+
+- {ref}`genai-02-mm-llm`
```

Comment:
i think this demo is removed, no?

Reply:
This is a valid link to a new tutorial: genai-02-model-monitor-llm.ipynb